queries.sql

```diff
@@ -1,3 +1,13 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
\ No newline at end of file
+SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%' || mon || '%';
```

Comment:
* Kindly use the proper quering synatx as `'%' || mon || '%'` would select all animals with the string `mon` when the requirement states select only animals that ends with the string `mon` please fix this 🙏